lib/katello/content/yum.rb

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
I would expect these content type registrations to live under `content_types` given there exists `content_type.rb` that loads them. Also, that seems a bit more explanatory.